test/exporter.js

```diff
@@ -0,0 +1,80 @@
+import test from 'ava';
+import sinon from 'sinon';
+
+import 'babel-register';
+import 'babel-polyfill';
+
+import Exporter from '../src/exporter';
+import { checksum } from '../src/helpers';
+
+let dbMock;
+let zipMock;
+let exporter;
+
+test.beforeEach(() => {
```

Comment:
There is `t.context` which allows to pass variables from `beforeEach` to a test https://github.com/avajs/ava#before--after-hooks

`ava` runs tests concurrently so setting global (in terms of tests) variables are no-no